libs/watt/project.json

```diff
@@ -36,17 +30,10 @@
       }
     },
     "build-storybook": {
-      "executor": "@storybook/angular:build-storybook",
-      "outputs": ["{options.outputDir}"],
+      "executor": "nx:run-commands",
       "options": {
-        "outputDir": "dist/designsystem",
-        "configDir": "libs/watt/.storybook",
-        "browserTarget": "watt:build-storybook",
-        "compodoc": false,
-        "stylePreprocessorOptions": {
-          "includePaths": ["libs/watt/src/lib/styles"]
-        },
-        "styles": ["libs/watt/.storybook/styles.scss"]
+        "cwd": "libs/watt",
+        "command": "storybook build --output-dir ../../dist/storybook/my-app"
```

Comment:
question: wonder whether the change to output directory from `designsystem` to `storybook/my-app` causes any deployment issues with Chromatic (after the PR is merged)?

Reply:
Think you are right, will looking to setting the same output path